xml/System.IO/StringReader.xml

```diff
@@ -829,7 +829,7 @@
 ## Remarks  
  This method overrides the <xref:System.IO.TextReader.ReadLine%2A?displayProperty=nameWithType> method.  
   
- A line is defined as a sequence of characters followed by a line feed ("\n"), a carriage return ("\r"), or a carriage return immediately followed by a line feed ("\r\n"). The string that is returned does not contain the terminating carriage return or line feed. The returned value is `null` if the end of the string has been reached.  
+ A line is defined as a sequence of characters followed by a line feed ("\n"), a carriage return ("\r"), a carriage return immediately followed by a line feed ("\r\n"), or the end-of-stream marker. The string that is returned does not contain the terminating carriage return or line feed. The returned value is `null` if the end-of-stream marker has been reached. That is to say, if there is nothing between the last new line and the end-of-stream marker, `null` will be returned.  
```

Comment:
- I find "the last new line" confusing. Better would be "the last line read"
- active voice is better than passive, so instead of "\`null\` will be returned" --> "the method returns \`null\`

Reply:
sounds good. Fixed!